2019-12-08 22:46:08,485 ERROR: Exception on /reset_password/eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJyZXNldF9wYXNzd29yZCI6NywiZXhwIjoxNTc1ODQ1NzEwLjQ1MzkyNzV9.e_AvPscOcVxLaPBMoGZwLSh4O-qmuX1u6PlOSqIpl7U [GET] [in C:\anaconda\envs\microblog\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "C:\anaconda\envs\microblog\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\anaconda\envs\microblog\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\anaconda\envs\microblog\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\anaconda\envs\microblog\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "C:\anaconda\envs\microblog\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\anaconda\envs\microblog\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\Martin\projects\microblog\app\routes.py", line 166, in reset_password
    form = ResetPasswordForm()
NameError: name 'ResetPasswordForm' is not defined
2019-12-08 22:50:53,203 INFO: Microblog startup [in D:\Martin\projects\microblog\app\__init__.py:49]
2019-12-08 22:51:02,062 ERROR: Exception on /reset_password/eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJyZXNldF9wYXNzd29yZCI6NywiZXhwIjoxNTc1ODQ1NzEwLjQ1MzkyNzV9.e_AvPscOcVxLaPBMoGZwLSh4O-qmuX1u6PlOSqIpl7U [GET] [in C:\anaconda\envs\microblog\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "C:\anaconda\envs\microblog\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\anaconda\envs\microblog\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\anaconda\envs\microblog\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\anaconda\envs\microblog\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "C:\anaconda\envs\microblog\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\anaconda\envs\microblog\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\Martin\projects\microblog\app\routes.py", line 167, in reset_password
    form = ResetPasswordForm()
NameError: name 'ResetPasswordForm' is not defined
2019-12-08 22:52:08,975 INFO: Microblog startup [in D:\Martin\projects\microblog\app\__init__.py:49]
